letsencrypt-auto-source/pieces/bootstrappers/gentoo_common.sh

```diff
@@ -9,15 +9,20 @@ BootstrapGentooCommon() {
     app-misc/ca-certificates
     virtual/pkgconfig"
 
+  local ASK_OPTION="--ask"
```

Comment:
We run `certbot-auto` with `/bin/sh` and `local` isn't POSIX compliant so it should be removed.